src/controller/content/content.service.ts

```diff
@@ -2307,5 +2324,67 @@ export class ContentService {
       return [{ message: err.message }]
     }
   }
+
+  async getAllQuizVariants(quizId: number): Promise<any> {
+    try {
+      const result = await db
```

Comment:
Incorporate a relationship mapping within the query. Avoid using the inner join  @sukanyamohanty143